app/controllers/v0/benefits_claims_controller.rb

```diff
@@ -5,6 +5,7 @@
 module V0
   class BenefitsClaimsController < ApplicationController
     before_action { authorize :lighthouse, :access? }
+    service_tag 'claims-shared'
```

Comment:
It looks like some of these are singular and some plural.

Reply:
"claim-status" is intentionally singular to reflect the name of the corresponding veteran-facing feature "claim status tool" 
"claims-shared" is intentionally plural reflecting the "claims" domain as a whole. 
The idea was to mirror how these are referred to in common usage among the teams. 

Unless you are saying I have an instance of `claim-shared` which I am not seeing.